src/Microsoft.Android.Runtime.NativeAOT/Android.Runtime.NativeAOT/JavaInteropRuntime.cs

```diff
@@ -2,7 +2,7 @@
 using Java.Interop;
 using System.Runtime.InteropServices;
 
-namespace NativeAOT;
+namespace Android.Runtime.NativeAOT;
```

Comment:
Given that these types are in a separate assembly, they could probably just stick with the `Android.Runtime` namespace.

Though I wonder if we should really be using `Android.*` for this anyway.  Maybe we should start using `Microsoft.Android.Runtime` as the namespace, to "mirror" the assembly name?

Reply:
I used `Microsoft.Android.Runtime`.